### PR TITLE
Added container_name to solve 'no such container' errors

### PR DESCRIPTION
**Pull request by drewchurch**
Updating example file and documentation regarding #2 as I ran into that as well.

### CODE REVIEW

**Comment by drewchurch** [comment]
@Kaiede for your review

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ In your `docker-compose.yml` file where you configure your minecraft server, you
       - /opt/bedrock/server:/server
 ```
 
-The service should always depend on all the bedrock servers listed in your `docker-compose.yml` file. We want to let the minecraft servers start before taking a backup on launch. 
+The service should always depend on all the bedrock servers listed in your `docker-compose.yml` file. We want to let the minecraft servers start before taking a backup on launch. You may also consider setting a `container_name` for each server that will be backed up as somes Docker will assign a different container name with a prefix.
```
Comment (Kaiede) [inline README.md:59]:
```suggestion
The service should always depend on all the bedrock servers listed in your `docker-compose.yml` file. We want to let the minecraft servers start before taking a backup on launch. It is recommended to set `container_name` for each server that will be backed up as docker-compose will assign one automatically if it isn’t set, and this simplifies configuring the backup service. 
```

**Review by Kaiede — CHANGES_REQUESTED** [review]
Mostly just a single suggestion, since you beat me to this update. I had some slightly different phrasing in mind already. Thanks for looking at this.

**Comment by drewchurch** [comment]
Looks great. Thanks!

**Pull request merged**